libs/mintycore/src/main/java/com/solanamobile/mintyfresh/mintycore/usecase/CarFileUseCase.kt

```diff
@@ -19,7 +19,7 @@ class CarFileUseCase @Inject constructor(
         // first get the media file info (metadata depends on the file cid)
         val mediaFile = File(imageFilePath)
         val mediaFileExtension = mediaFile.extension
-        val mediaMimeType = Files.probeContentType(mediaFile.toPath())
+        val mediaMimeType = Files.probeContentType(mediaFile.toPath()) ?: "image/jpeg"
```

Comment:
A bit skeptical about harcoding jpeg here. It could be png as well. 

Media Content provider should already be filtering images, so this should be easier replace with a different check from content provider API? Not sure about this though

Reply:
This happened when I shared from a different app. I *think* is as unfortunately lame as the sending app didn't specify a filename on the shared file. I'm not opposed to making this more robust, but i'm not quite sure how we should do it.

Reply:
cant you do "image/*" to cover all image types? would that be an acceptable option here? forgive me I forget all the details that went into this bit and if that will work or not.